docs/content/docs/install/_index.md

```diff
@@ -61,7 +87,7 @@ To install OSM on OpenShift:
     ```shell
     osm install --set="OpenServiceMesh.enablePrivilegedInitContainer=true"
     ```
-    - If you have already installed OSM without enabling privileged init containers, set `enable_privileged_init_container` to `true` in the [OSM ConfigMap](../osm_config_map.md) and restart any pods in the mesh.
+    - If you have already installed OSM without enabling privileged init containers, set `enable_privileged_init_container` to `true` in the [OSM ConfigMap](../osm_config_map) and restart any pods in the mesh.
```

Comment:
I thought we need to use the file extension (`../osm_config_map.md`) and use an alias of `osm_config_map.md` for the configmap doc so it renders fine on the website.

Why is the extension being removed?

Reply:
I am referring to how internal links are to be referenced: https://github.com/openservicemesh/osm/tree/main/docs#internal-links

Reply:
@ksubrmnn , based on discussions offline, we don't need to have this link navigable on github. My original comment can be ignored.